lib/rust/Kconfig

```diff
@@ -0,0 +1,13 @@
+# Rust configuration options
+#
+# Copyright (c) 2024 Linaro LTD
+# SPDX-License-Identifier: Apache-2.0
+
+menu "Rust Language Support"
+
+config RUST
+	bool "Rust support for the application"
+	help
```

Comment:
I think `select EXPERIMENTAL` should be added :)

Reply:
Done